addons/web_editor/static/src/scss/web_editor.backend.scss

```diff
@@ -1,7 +1,6 @@
 .oe_form_field_html {
     position: relative;
     word-wrap: break-word;
-    overflow: hidden;
```

Comment:
That is risky. You want to test this very carefuly in lots of cases, in iframe, in modals, etc. Especially with content that would overflow it like huge contents or controls, dropdowns and such.

Reply:
I'll look into it.

Reply:
Nicolas confirmed he added it for a use case that doesn't exist anymore.